From 7d6269fc4891807896552ebdde408975b76e97a0 Mon Sep 17 00:00:00 2001
From: Daniel Wagner <dwagner@suse.de>
Date: Fri, 15 Jan 2021 19:37:07 +0100
Subject: [PATCH] pip_stress: De-constify prio_min

This variables is updated by the result of
sched_get_priority_min(). Hence, it can't be const.

Signed-off-by: Daniel Wagner <dwagner@suse.de>
Signed-off-by: John Kacur <jkacur@redhat.com>

---
 src/pi_tests/pip_stress.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pi_tests/pip_stress.c b/src/pi_tests/pip_stress.c
index e6cf0ce..a0c3b89 100644
--- a/src/pi_tests/pip_stress.c
+++ b/src/pi_tests/pip_stress.c
@@ -77,7 +77,7 @@ struct State {
 struct State *statep;
 
 const int policy = SCHED_FIFO;
-const int prio_min;	/* Initialized for the minimum priority of policy */
+static int prio_min;	/* Initialized for the minimum priority of policy */
 
 struct option long_options[] = {
     { "usleep", required_argument, 0, 0 },
